app/shared/user-agent-client.js

```diff
@@ -0,0 +1,134 @@
+/*
+ Copyright 2016 Mozilla
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ this file except in compliance with the License. You may obtain a copy of the
+ License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed
+ under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ specific language governing permissions and limitations under the License.
+ */
+
+import path from 'path';
+import EventEmitter from 'events';
+import WebSocket from 'ws';
+import request from 'request';
```

Comment:
We have isomorphic-fetch, do we also need request? We should have one or the other